code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -216,6 +218,8 @@
 		toxloss = 0
 	else
 		amount = amount*species.toxins_mod
+		if(stats.getPerk(PERK_BLOOD_OF_LEAD))
```

Comment:
This would make chems, such as dylovene, half as effective at healing you.

Is this intended?

Reply:
Same as above.